app/src/androidTest/java/com/nervousfish/nervousfish/test/MainSteps.java

```diff
@@ -153,6 +170,28 @@ public void iShouldGoToSettingsScreen() {
         intended(hasComponent(SettingsActivity.class.getName()));
     }
 
+    @Then("^the app shouldn't crash because of Bluetooth$")
+    public void theAppShouldCrashBecauseOfBluetooth() {
+        assertTrue(true);
```

Comment:
No, the problem is the Android popup requesting the user to let the device be searchable for X amounts of seconds. Espresso is (afaik) not able to click on that popup and as long as its there we can't verify anything. Resulting in the test case error due to a timeout 😕